.github/workflows/cargo-build-lint-and-test-on-pr-and-push.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2022 The ModelarDB Contributors
```

Comment:
If the GitHub related files have the copyright now, should we also add it to the `.gitignore` file?

Reply:
It is not something I have seen before, e.g., the [Apache Flink](https://github.com/apache/flink/blob/master/.gitignore), [Apache Spark](https://github.com/apache/spark/blob/master/.gitignore), and [Django](https://github.com/django/django/blob/main/.gitignore) does not include a license header in the `.gitignore` file. However, [Apache Arrow](https://github.com/apache/arrow/blob/main/.gitignore) mono-repository and [Apache Arrow DataFusion](https://github.com/apache/arrow-datafusion/blob/main/.gitignore) does. Although, the repository with the [Rust version of Apache Arrow does not](https://github.com/apache/arrow-rs/blob/master/.gitignore). So even among Apache projects there does not seem to be any consistency, but as it is simpler if we just add the header to all files that are not automatically generated compared to a subset, I have added the header to the `.gitignore` file.